### ISSUE TITLE
Order does not exist

### ISSUE THREAD

**Issue by nfacha** [ccxt/binance-trade-bot#13]
First order is executed but than it gives a "Order does not exist" causing it to have no balance to continue

```
2021-01-30 22:46:04,838 - crypto_trader_logger - INFO - Jumping from DOGE to XLM
2021-01-30 22:46:07,051 - crypto_trader_logger - INFO - order
2021-01-30 22:46:07,051 - crypto_trader_logger - INFO - {'symbol': 'DOGEUSDT', 'orderId': 167939712, 'orderListId': -1, 'clientOrderId': '52j6rP6AUqajIprbUEd3LG', 'transactTime': 1612043166940, 'price': '0.00000000', 'origQty': '2100.00000000', 'executedQty': '2100.00000000', 'cummulativeQuoteQty': '56.86191000', 'status': 'FILLED', 'timeInForce': 'GTC', 'type': 'MARKET', 'side': 'SELL', 'fills': [{'price': '0.02707710', 'qty': '2100.00000000', 'commission': '0.05686191', 'commissionAsset': 'USDT', 'tradeId': 19356651}]}
2021-01-30 22:46:09,291 - crypto_trader_logger - INFO - APIError(code=-2013): Order does not exist.
2021-01-30 22:46:12,142 - crypto_trader_logger - INFO - Jumping from DOGE to XLM
2021-01-30 22:46:14,373 - crypto_trader_logger - INFO - APIError(code=-1013): Invalid quantity.
```

**Comment by ivanyyw** [comment]
I used to get this APIError(code=-2013): Order does not exist. error and it causes the whole script run into infinite loop "Failed to Buy/Sell. Trying Again."

I wonder is there a better way to catch this API error code and re-try to check after "seconds" and continue script.. Since Binance might takes longer time to get update the order id..

**Comment by nfacha** [comment]
> I used to get this APIError(code=-2013): Order does not exist. error and it causes the whole script run into infinite loop "Failed to Buy/Sell. Trying Again."
> 
> I wonder is there a better way to catch this API error code and re-try to check after "seconds" and continue script.. Since Binance might takes longer time to get update the order id..

In theory that was the fix applied for #9, but the issue still persists

**Comment by ivanyyw** [comment]
> > I used to get this APIError(code=-2013): Order does not exist. error and it causes the whole script run into infinite loop "Failed to Buy/Sell. Trying Again."
> > I wonder is there a better way to catch this API error code and re-try to check after "seconds" and continue script.. Since Binance might takes longer time to get update the order id..
> 
> In theory that was the fix applied for #9, but the issue still persists

I found a way to eliminate this issue, but i'm not sure whether its best approach.
I noticed the triggering happen whenever client.get_order() calling.

If the binance server was slow updating the order id, the API return with -2013 error.
By normal, this exception was caught in exceptions.py and returned to crypto_trading.py with Trying loop.

By using the same exceptions.py to be intercept at our code:

```
from binance.exceptions import BinanceAPIException

try:
     stat = client.get_order(symbol=alt_symbol+crypto_symbol, orderId=order[u'orderId'])
except BinanceAPIException as e:
     logger.info(e)
     time.sleep(10)
```

If API returns with APIError(code=-2013): it will sleep for 10 before trying get_order again.

Hope it helps!
